### ISSUE TITLE
No resolver for typescript with ts-jest

### ISSUE THREAD

**Issue by cojack** [sequelize/umzug#387]
Version: 3.0.0-beta.8
Storage: SequelizeStorage
Sequelize: 6.1.1
Node: 15.0.1
Dev setup: Typescript@3.9.6

When I try to run test with jest with preset: ts-jest I've got a error back: 

```
No resolver specified for file /home/cojack/Projects/xxx/xxx-sequelize/src/__tests__/migrations/1589746665497-example.ts. You can use the default resolver with typescript files by adding `ts-node` as a dependency and calling `require('ts-node/register')` at the program entrypoint before running migrations. See docs for guidance on how to write a custom resolver.
```

So I have no clue, how this should working. Any advice? I've tried to add a `require('ts-node/register');` at the top of the entry point, where it was a jest.config.js file, also I tried to run jest with configuration like:

```
node -r ts-node/register ./node_modules/.bin/jest ...
```

without success, also tried to add this require into the test file, also without success, same thing happen.

Also repository doesn't provide any example of tests that works that way, only **README** with hidden section explains how it should works, but it doesn't. 

So my question is: does it works? And if so, how I should configure it to works that way.

Regards.

**Comment by cojack** [comment]
Related to the ts-jest https://github.com/kulshekhar/ts-jest/issues/2144

**Comment by mmkal** [comment]
@cojack thanks for the report. Just to check, do your migrations work ok when running normally (i.e. outside of jest)?

Either way, it shouldn't be too hard to find a workaround which detects ts-jest and lets you require typescript from that environment too.

In the meantime did you try adding a custom resolver that works like the default one but skips file extension validation? It'd be something like:

```typescript
migrations: {
  glob: 'a/b/*.ts',
  resolve: params => ({ ...params, ...require (params.path!) }),
}
```
```

**Comment by mmkal** [comment]
@cojack fix released in 3.0.0-beta.9 (`npm install umzug@beta`), let me know if you have problems with it.

**Comment by cojack** [comment]
@mmkal yes, now it works, thanks!
